packages/benchmark/package.json

```diff
@@ -4,6 +4,11 @@
   "version": "0.15.0",
   "description": "Benchmark utils",
   "keywords": [],
+  "author": {
+    "name": "Alex Chugaev",
+    "email": "achugaev93@gmail.com",
+    "url": "https://github.com/avchugaev"
+  },
```

Comment:
you can add your name in this package too

Reply:
sure, added in the last commit:ok_hand: